.github/workflows/node-release.yml

```diff
@@ -119,6 +120,7 @@ jobs:
           node-version-file: '.nvmrc'
 
       - name: npm ci
+        working-directory: platform/node
```

Comment:
I am unsure if this npm ci is needed or not. in the node-ci it does not included this working-directory, so it seems it would behave differently... but that didn't seem to make a difference in the build so maybe this step isnt needed?

Reply:
Thinking now , the npm dev dependencies might be needed for the tests, which don't run in the release...so maybe that explains why it didn't make a differenct.

I think at minimum this `working-directory: platform/node` line I added should be removed